src/components/HomeView.vue

```diff
@@ -46,3 +84,50 @@ export default {
   },
 };
 </script>
+<!-- TODO: remove unnessesary comments -->
```

Comment:
all these todos need to be addressed or have an issue number added before we merge this pr